bin/ci-tests.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# print a command and execute it
+show() {
```

Comment:
Revert this please, bash has `set -x` to echo commands, and `set +x` to stop. The changes in this file are not required, lets leave CI improvements for separate PRs, so we don't modify release code and test code at the same time